src/Validator/Constraints/PhoneNumberValidator.php

```diff
@@ -126,6 +137,36 @@ public function validate($value, Constraint $constraint)
         }
     }
 
+    private function getRegion(Constraint $constraint): ?string
+    {
+        if (null === $path = $constraint->regionPath) {
+            return $constraint->defaultRegion ?? $this->defaultRegion;
+        }
+
+        $object = $this->context->getObject();
+        if (null === $object) {
+            throw new \LogicException('The current validation does not concern an object');
+        }
+
+        try {
+            return $this->getPropertyAccessor()->getValue($object, $path);
```

Comment:
How should we behave if the return here is null ? ignore the default values or take them again?